psychopy_eyetracker_sr_research/sr_research/eyelink/eyetracker.py

```diff
@@ -145,6 +145,11 @@ def __init__(self, *args, **kwargs):
                     EyeTracker._keyboard = dev
 
         try:
+            # Add additional EyeLink command functions as options for the overloaded
```

Comment:
this was moved to the top because the functions need to be available during the connection initialization